src/test/resources/transcriptomics-denovo-workflow.ga

```diff
@@ -0,0 +1,2234 @@
+{
+  "a_galaxy_workflow": "true",
+  "annotation": "De novo transcriptome reconstruction with RNA-Seq",
+  "format-version": "0.1",
+  "name": "Transcriptomics Denovo Workflow",
+  "steps": {
+    "0": {
+      "annotation": "",
+      "content_id": null,
+      "errors": null,
+      "id": 0,
+      "input_connections": {},
+      "inputs": [
+        {
+          "description": "",
+          "name": "G1E_rep1_forward_read"
+        }
+      ],
+      "label": null,
+      "name": "Input dataset",
+      "outputs": [],
+      "position": {
+        "left": 10,
+        "top": 10
+      },
+      "tool_id": null,
+      "tool_state": "{\"name\": \"G1E_rep1_forward_read\"}",
+      "tool_version": null,
+      "type": "data_input",
+      "uuid": "cbc6590c-d6de-4abf-b53f-6c83b4ab4120",
+      "workflow_outputs": []
+    },
+    "1": {
+      "annotation": "",
+      "content_id": null,
+      "errors": null,
+      "id": 1,
+      "input_connections": {},
+      "inputs": [
+        {
+          "description": "",
+          "name": "G1E_rep1_reverse_read"
+        }
+      ],
+      "label": null,
+      "name": "Input dataset",
+      "outputs": [],
+      "position": {
+        "left": 10,
+        "top": 130
+      },
+      "tool_id": null,
+      "tool_state": "{\"name\": \"G1E_rep1_reverse_read\"}",
+      "tool_version": null,
+      "type": "data_input",
+      "uuid": "c2ba4aba-060a-40ba-88d6-e6fef8a1f2eb",
+      "workflow_outputs": []
+    },
+    "10": {
+      "annotation": "",
+      "content_id": "toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.72",
```

Comment:
@jmchilton any idea why this workflow has readable content_ids but the workflow above has mostly null content_ids?
Thinking it might be more interesting for the tool tab

Reply:
content_id is essentially and alias for tool_id - it is empty for inputs and such, the above workflow uses a built in tool (cat) so it has one content_id populated. This content_id seems more descriptive because the tool is a tool shed tool and so the id is URI instead of a simpler string.